spec/factories/external_post.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :external_post do
+    title 'Example Discussion'
+    body_markdown 'Example body. So **strong**.'
+    user_display_name 'Mister Test'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.